### DISCUSSION TITLE
Actividad Docker - Grupo 5

### DISCUSSION THREAD

**Discussion by DanielUTN** [frlp-utn-ingsoft/catedra#108] [Show and tell]
https://github.com/DanielUTN/vetsoft

se trabajo en una rama nuevo branchDocker 
y luego hicimos merge al main.
Saludos

**Comment by rodriguezemautn** [comment]
Correcciones :
- el archivo Dockerfile se escribe con inicial mayuscula
- en el repo no esta el archivo -env-exmaple
- verifiquen el orden  de las sugerencias del README.md
-no despliega el contenedor
-no debe ser necesario ejecutar docker exec cambiar el orden en lienas CMD o realizar un script.sh copiarlo al interior del contenedor y utilizar ENTRYPOINT

Puntos aprobados:
* Construye la imagen
* Buen tamaño de imagen!
$ docker images -a
REPOSITORY    TAG       IMAGE ID       CREATED         SIZE
vetsoft-app   1.0       2ad00620a4fb   4 minutes ago   307MB

**Reply by JoseLlontop** [reply]
Ya están hechas las correcciones.
